src/components/modals/ConditionModal.tsx

```diff
@@ -227,26 +231,26 @@ const Component: React.FC<Props> = (props) => {
             return
         }
 
-        const condition: CLM.Condition = {
+        const cond: CLM.Condition = {
```

Comment:
I can't imagine what warning this could have been from.  What is a `cond`? Would change it back

Reply:
This is to avoid shadowing another `condition` variable in the same scope @ https://github.com/microsoft/ConversationLearner-UI/blob/c47043ce77a090c907e5e84f7ee6f5dc35c33d25/src/components/modals/ConditionModal.tsx#L190

Reply:
Then make it `conditionFromState` as opposed to other `condition` from props. I think it is better to add information to differentiate then take away and rising confusion or ambiguity.  Also, think the variable name should indicate what is in in isolation without relying on surrounding context.

Reply:
Done.  Note that the context here is a 10-line function, including whitespace.